test/integration_test/fafb_matrix_test.go

```diff
@@ -47,12 +46,14 @@ var (
 	}
 )
 
-var flashArrayPositiveInstallTests = []PureTestrailCase{
+// TODO: add readable test names
+var flashArrayPositiveInstallTests = []TestCase{
 	//// C55130: https://portworx.testrail.net/index.php?/cases/view/55130
 	// Install PX through operator with internal KVDB and journal drive on FlashArray
 	{
-		TestCase: TestCase{
-			TestrailCaseIDs: []string{"C55130"},
+		TestName:        "C55130",
```

Comment:
@Pure-AdamuKaapan in future let's add user readable names to existing test cases to better understand the intention.

Reply:
Sounds good to me, I'll also go back and revise these names next time I work on this code